src/python_testing/TC_ACE_1_2.py

```diff
@@ -0,0 +1,272 @@
+#
+#    Copyright (c) 2022 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+import queue
+
+import chip.clusters as Clusters
+from chip.clusters import ClusterObjects as ClustersObjects
```

Comment:
Is this subtle rename intended or a typo?

Reply:
ha, no - not intended. Copied it from a another script with this error.